ddtrace/contrib/flask/patch.py

```diff
@@ -636,6 +638,9 @@ def _set_request_tags(span):
 
         span.set_tag_str(COMPONENT, config.flask.integration_name)
 
+        if span.name.split(".")[-1] == "request":
```

Comment:
not-blocking: We should not be calling this function when the span is not the flask request. This is a result of how the wsgi middleware is used to update the span resource.